internal/store/store.go

```diff
@@ -69,6 +75,14 @@ func (s *DataStore) Assessment() Assessment {
 	return s.assessment
 }
 
+func (s *DataStore) Snapshot() Snapshot {
+	return s.snapshot
+}
+
+func (s *DataStore) RiverJob() RiverJob {
+	return s.riverJob
```

Comment:
dont't forget to close the connection to db in Close()

Reply:
When the main process exits, you need to close the pool you created for river client. Or at least call riverClient.Close() if available.